src/client/css/global.css

```diff
@@ -402,12 +405,23 @@ button.clear-button {
 .button-container {
   display: flex;
 }
+
 .button-container button {
   margin-right: 10px;
   width: auto;
   min-width: 16vh;
 }
 
+@media (max-width: 768px) {
+  .button-container {
+    flex-direction: column;
+  }
+  .button-container button {
+    width: 100%;
+    min-width: 16vh;
```

Comment:
ah, missed this last time — my instinct tells me this would better as either a fixed `px` width or at least `vw` to be a ratio of the screen width